tests/InsertionsClientTest/InsertionApiTest.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.Net.Insertions.Api;
+using Microsoft.Net.Insertions.Api.Providers;
+using Microsoft.Net.Insertions.Common.Constants;
+using Microsoft.Net.Insertions.Models;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace InsertionsClientTest
+{
+    [TestClass]
+    public class InsertionApiTest
+    {
+		[TestMethod]
```

Comment:
Wrong indentation?

Reply:
Good catch.  The IDE didn't show the inconsistent indentation. I couldn't reproduce with tabs.  I think I resolved it by copying the spacing of other test methods.